src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1445,23 +1572,23 @@ private TaskDialogEventArgs DisplayEngineFailureMessage(Exception exception)
         /// <returns> true if the file must be opened and false otherwise </returns>
         private bool DisplayFutureFileMessage(string fullFilePath, Version fileVersion, Version currVersion)
         {
-            var fileVer = ((fileVersion != null) ? fileVersion.ToString() : Properties.Resources.UnknownVersion);
-            var currVer = ((currVersion != null) ? currVersion.ToString() : Properties.Resources.UnknownVersion);
+            var fileVer = ((fileVersion != null) ? fileVersion.ToString() : Resources.UnknownVersion);
+            var currVer = ((currVersion != null) ? currVersion.ToString() : Resources.UnknownVersion);
 
             InstrumentationLogger.LogPiiInfo("FutureFileMessage", fullFilePath +
                 " :: fileVersion:" + fileVer + " :: currVersion:" + currVer);
 
-            string summary = Properties.Resources.FutureFileSummary;
-            var description = string.Format(Properties.Resources.FutureFileDescription, fullFilePath, fileVersion, currVersion);
+            string summary = Resources.FutureFileSummary;
+            var description = String.Format(Resources.FutureFileDescription, fullFilePath, fileVersion, currVersion);
 
             const string imageUri = "/DynamoCoreWpf;component/UI/Images/task_dialog_future_file.png";
             var args = new TaskDialogEventArgs(
                 new Uri(imageUri, UriKind.Relative),
-                Properties.Resources.FutureFileTitle, summary, description) { ClickedButtonId = (int)ButtonId.Cancel };
+                Resources.FutureFileTitle, summary, description) { ClickedButtonId = (int)ButtonId.Cancel };
 
-            args.AddRightAlignedButton((int)ButtonId.Cancel, Properties.Resources.CancelButton);
-            args.AddRightAlignedButton((int)ButtonId.DownloadLatest, Properties.Resources.DownloadLatestButton);
-            args.AddRightAlignedButton((int)ButtonId.Proceed, Properties.Resources.ProceedButton);
+            args.AddRightAlignedButton((int)ButtonId.Cancel, Resources.CancelButton);
+            args.AddRightAlignedButton((int)ButtonId.DownloadLatest, Resources.DownloadLatestButton);
+            args.AddRightAlignedButton((int)ButtonId.Proceed, Resources.ProceedButton);
```

Comment:
Hi @aparajit-pratap, I'm stopping the review now because there are too many formatting related changes. It's not easy to review, please take them out to a separate PR (and let me know, I'll merge it right away), and then continue here. Thanks!